### PR TITLE
BaSP-MR-54: Class, List/Delete Table

### PR DESCRIPTION
**Pull request by ale-schmidt**
List table with delete functionality as well as a modal to confirm delete action created.

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Review by lorenzomauro59 — APPROVED** [review]
Everything looks well!

File: src/Components/Classes/Modal/index.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import styles from './modal.module.css';
+
+const Modal = (props) => {
+  if (!props.showModal) {
+    return null;
+  }
+
+  const CloseModal = () => {
```
Comment (LucianoAlarcon) [inline src/Components/Classes/Modal/index.jsx]: Please renam this function to `closeMolda`

File: src/Components/Classes/index.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import { useState, useEffect } from 'react';
+import Modal from './Modal';
+import styles from './classes.module.css';
+// import Form from './Form';
```
Comment (LucianoAlarcon) [inline src/Components/Classes/index.jsx]: Delete this comment

**Review by LucianoAlarcon — CHANGES_REQUESTED** [review]

**Review by LucianoAlarcon — APPROVED** [review]

**Comment by ale-schmidt** [comment]
[[Class] List/Delete table](https://trello.com/c/em2OKyQY/54-class-list-delete-table)

**Pull request merged**